packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -180,14 +180,14 @@ export function getPropGetters<TItem, TEvent, TMouseEvent, TKeyboardEvent>({
     const { inputElement, maxLength = 512, ...rest } = providedProps || {};
 
     return {
-      'aria-autocomplete': props.enableCompletion ? 'both' : 'list',
+      'aria-autocomplete': 'both',
```

Comment:
> For a textbox with the aria-autocomplete attribute set to either inline or both, authors should ensure that any auto-completed text is selected, so the user can type over it.

https://accessibilityresources.org/aria-autocomplete

I don't see any sites doing this though, so I don't think it really makes sense like that

Reply:
From https://www.digitala11y.com/aria-autocomplete-properties/:

> It distinguishes between two models: the inline model (aria-autocomplete=”inline”) that presents a value completion prediction inside the text input and the list model (aria-autocomplete=”list”) that presents a collection of possible values in a separate element that pops up adjacent to the text input. It is possible for an input to offer both models at the same time (aria-autocomplete=”both”).

Since we provide both the completion in the input _and_ suggestions in the dropdown, I believe `both` is the right value for us.